mgmt/rest/server.go

```diff
@@ -92,6 +93,15 @@ func New(cfg *Config) (*Server, error) {
 		negroni.HandlerFunc(s.authMiddleware),
 	)
 	s.r = httprouter.New()
+
+	// Deals with CORS
+	c := cors.New(cors.Options{
+		AllowedOrigins: []string{"*"},
+		AllowedMethods: []string{"GET, POST, DELETE, PUT, PATCH, OPTIONS"},
```

Comment:
Not all of the HTTP methods were allowed previously. I believe it would be the safest to keep the list of allowed methods to a required minimum.
`PATCH` - I didn't find it declared for any of REST controllers. Are you sure we need to allow it? 
`OPTIONS` - the same as for `PATCH`. It's not needed for pre-flight requests. Negroni allows it implicitly only for the purpose of CORS. Furthermore, if it's needed for Snap API, it should be enabled with a flag `OptionsPassthrough`.